frame/dvm/src/lib.rs

```diff
@@ -435,6 +441,28 @@ impl<T: Config> Module<T> {
 			used_gas.unique_saturated_into(),
 		))
 		.into())
+
+	}
+
+    pub fn raw_call(
```

Comment:
Why not use `do_transact` to do this `call`?

Reply:
`do_transact` will check nonce, check gas price, and save pending. I think it's not suitable here.

Reply:
There is another way can achieve this. https://github.com/darwinia-network/darwinia-common/blob/master/bin/node/runtime/pangolin/src/lib.rs#L798

The advantage of your implementation is that the external crate just only interacts directly with the dvm pallet, which I think is acceptable.

Reply:
yes, `do_call` seems better.